### ISSUE TITLE
Include the "why" of this settings?

### ISSUE THREAD

**Issue by Bartmax** [reactiveui/documentation#32]
I'm starting my Rx journey, and while I see this type of suggestion:
https://github.com/reactiveui/documentation/edit/master/en/getting-started/visual-studio-settings.md
Disable `Just my code` and `Stop on first chance exceptions`. 

I know what those settings are, what I do not know is **why** you suggest to set into a specific value. I think that will be great for the documentation. To educate the reader instead of tell them to do this and that.

**Comment by ghuntley** [comment]
Agreed; check your email @Bartmax for invitation to Slack and GitHub. I know it doesn't help in this specific case but you now have the ability to edit the documentation directly and contribute nuggets of knowledge as you learn.
